### DISCUSSION TITLE
Weird page behaviour when site uses -, but you use _ in the adress.

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2226] [Q&A]
I've noticed a somewhat weird or odd behaviour and can't figure out if it would be a MkDocs problem, Materials problem or similar.

I noticed, that when you have a page/file name with a hyphen (`-`) but then use a underscore instead (https://example.com/the-page becomes https://example.com/the_page) that the entire page becomes broken.
No 404 error page, no redirect to the actual one, just a white bg with some malformed text and similar (See image).

Here's an example:
The site https://docs.purrbot.site/bot/welcome-images/ becomes broken when you try to connect to https://docs.purrbot.site/bot/welcome_images/ instead.

I again can't tell if that is something caused by either MkDocs, Materials for MkDocs or something else.

Screenshot:
![image](https://user-images.githubusercontent.com/11576465/105231220-d566ad80-5b66-11eb-9bb2-994179860b83.png)

**Comment by squidfunk** [comment]
Likely related to your deployment (e.g. web server config); doesn't happen on the official docs which will correctly 404:
https://squidfunk.github.io/mkdocs-material/creating_your_site/

Looks like broken asset paths.

**Reply by Andre601** [reply]
Not sure how I could change/alter this myself.

I do have this setup for deploying the docs.
https://github.com/purrbot-site/Docs/blob/master/.github/workflows/deploy.yml
